### PR TITLE
CVS-41121 onnx-support

### PR DESCRIPTION
**Pull request by mgumowsk**

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
LGTM

File: src/modelinstance.cpp [hunk 1]
```diff
@@ -226,7 +226,7 @@ std::unique_ptr<InferenceEngine::CNNNetwork> ModelInstance::loadOVCNNNetworkPtr(
 }
 
 Status ModelInstance::loadOVCNNNetwork() {
-    auto& modelFile = modelFiles[".xml"];
+    auto& modelFile = modelFiles[0];
```
Comment (mzegla) [inline src/modelinstance.cpp:229]: For IR format. Are we sure **.xml** file will be the first element of this vector every time and not the **.bin** one?

Comment (mgumowsk) [reply src/modelinstance.cpp:229]: yes, see OV_MODEL_FILES_EXTENSIONS in modelinstance.hpp

Comment (mzegla) [reply src/modelinstance.cpp:229]: oh, I see, it's all good in this case

**Review by mzegla — APPROVED** [review]

**Pull request merged**